client/src/main/java/com/influxdb/client/TaskQuery.java

```diff
@@ -0,0 +1,112 @@
+package com.influxdb.client;
+
+import javax.annotation.Nullable;
+
+public class TaskQuery {
+
+    /**
+     * Returns task with a specific name. (optional)
+     */
+    @Nullable
+    private String name;
+
+    /**
+     * Return tasks after a specified ID. (optional)
+     */
+    @Nullable
+    private String after;
+
+    /**
+     * Filter tasks to a specific user ID. (optional)
+     */
+    @Nullable
+    private String user;
+
+    /**
+     * Filter tasks to a specific organization name. (optional)
+     */
+    @Nullable
+    private String org;
+
+    /**
+     * Filter tasks to a specific organization ID. (optional)
+     */
+    @Nullable
+    private String orgID;
+
+    /**
+     * Filter tasks by a status--\&quot;inactive\&quot; or \&quot;active\&quot;. (optional)
+     */
+    @Nullable
+    private String status;
+
+    /**
+     * The number of tasks to return (optional, default to 100)
```

Comment:
Fix `JavaDoc` formatting:
```suggestion
     * The number of tasks to return. (optional, default to 100)
```